docs/home/contribute/participating.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Participating in SIG-DOCS
+---
+
+{% capture overview %}
+
+SIG-DOCS is one of the [special interest groups](https://github.com/kubernetes/community/blob/master/sig-list.md) within the Kubernetes project, focused on writing, updating, and maintaining the documentation for Kubernetes as a whole.
+
+{% endcapture %}
+
+{% capture body %}
+
+SIG Docs welcomes content and reviews from all contributors. Anyone can open a pull request (PR), and anyone is welcome to comment on content or pull requests in progress.
+
+Within the Kubernetes project, you may also become a member, reviewer, or approver.
+These roles confer additional privileges and responsibilities when it comes to approving and committing changes.
+See [community-membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for more information on how membership works within the Kubernetes community.
+
+## Roles and Responsibilities
+
+- Members
+
+Any member of the [Kubernetes organization](https://github.com/kubernetes) can review a pull request, and SIG Docs team members frequently request reviews from other SIG members for technical accuracy. 
+You can indicate your approval by adding a comment of `/lgtm` to a pull request.
+
```

Comment:
It's true that any member of the Kubernetes organization can review a pull request. But it's also true that people outside of the Kubernetes organization can review pull requests. I think we should make that clear.